src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -100,19 +100,13 @@ public ShooterSubsystem() {
     flywheelPID = flywheelController.getPIDCtrl();
     flywheelEncoder = flywheelController.getEncoder();
     flywheelController.setInverted(true);
-    flywheel2Controller.follow(flywheelController);
+    flywheel2Controller.follow(flywheelController, true);
     // Initializes the SparkMAX for the hood TODO: Set this up when possible
     /*hoodController = new  MotorController("Hood", Constants.hoodID);
     hoodPID = hoodController.getPID();
     hoodEncoder = hoodController.getEncoder();*/
     // Initializes the SparkMAX for the cargo stopper
     stopperController = new MotorController("Shooter Cargo", Constants.Shooter.shooterCargoID);
-    // Initializes PID for the flywheel
-    flywheelPID.setP(5e-4);
-    flywheelPID.setI(6e-7);
-    flywheelPID.setIMaxAccum(0.9, 0);
-    flywheelPID.setD(0.0);
-    flywheelPID.setOutputRange(0, 1);
```

Comment:
Where do these PID values get set instead?

Reply:
You set defaults in the Shuffleboard entries, but updatePid has to be executed manually before the PIDs get setup. #144 Will provide a better solution for this with a new MotorController constructor

Reply:
Should I just add the PID sets to the constructor to pass this PR, and use #144 when it exists?

Reply:
You don't have to fix it for this PR, just be aware of it for testing and fix it up once #144 is in

Reply:
Alright, got a little bit into the weeds here, but I reworked the shooter subsystem's use of PID and added FF support to the motor controller class